zerver/actions/invites.py

```diff
@@ -184,7 +184,7 @@ def check_invite_limit(realm: Realm, num_invitees: int) -> None:
 @transaction.atomic(durable=True)
 def do_invite_users(
     user_profile: UserProfile,
-    invitee_emails: Collection[str],
+    invitee_emails: Collection[tuple[str, str]],
```

Comment:
Can this and similar variables below be renamed to make it clear that they are tuple containing email and name?

Reply:
Or maybe `invitee_names_and_emails`?